java/src/test/java/com/google/appengine/tools/pipeline/PipelineSetupExtensions.java

```diff
@@ -0,0 +1,89 @@
+package com.google.appengine.tools.pipeline;
+
+import com.google.appengine.tools.pipeline.impl.PipelineManager;
+import com.google.appengine.tools.pipeline.impl.backend.AppEngineBackEnd;
+import com.google.appengine.tools.pipeline.impl.backend.AppEngineTaskQueue;
+import com.google.cloud.datastore.Datastore;
+import org.junit.jupiter.api.extension.*;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.util.Arrays;
+import java.util.List;
+
+@Target({ ElementType.TYPE, ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@ExtendWith({ DatastoreExtension.class, PipelineComponentsExtension.class, PipelineComponentsExtension.ParameterResolver.class })
+public @interface PipelineSetupExtensions {
```

Comment:
composes all the extensions we want generally for pipeline tests. eventually, can replace `PipelineTest` class